src/map/lua/lua_baseentity.cpp

```diff
@@ -11229,6 +11229,39 @@ inline int32 CLuaBaseEntity::getWeaponDmgRank(lua_State *L)
     return 1;
 }
 
+/************************************************************************
+*  Function: getWeaponID()
+*  Purpose : Returns the item ID for the weapon in the passed in SLOT
+*  Example : player:getWeaponID()
+*  Notes   : Primarily used in REM weapon scripts to determine which version of the weapon is being used
+************************************************************************/
+
+inline int32 CLuaBaseEntity::getWeaponID(lua_State* L)
```

Comment:
what does this do that getEquipID doesn't?

Reply:
Nothing. I didn't find getEquipID right away. Will removed and swap over.